analysis_configs/t_channel_lost_lepton_control_region.py

```diff
@@ -18,38 +19,32 @@ def process(events, cut_flow, year, primary_dataset="", pn_tagger=False, **kwarg
         events = sequences.remove_primary_dataset_overlap(events, year, primary_dataset)
         skimmer_utils.update_cut_flow(cut_flow, "PrimaryDatasetOvelap", events)
 
+    # Add new branches at the start so that the defined branches can be used thereafter
+    events = sequences.add_good_ak8_jet_branch(events)
+    events = sequences.add_good_ak4_jet_branch(events)
```

Comment:
Wouldn't it be good to replace
good_ak4_jets = events.Jets[obj.is_good_ak4_jet(events.Jets)]
with good_ak4_jets = events.Jets[events.Jets.isGood]
for consistency?

Reply:
Yes that's better indeed!